universal-login-react/src/ui/styles/themes/UniLogin/noticeThemeUniLogin.sass

```diff
@@ -0,0 +1,11 @@
+.unilogin-theme-unilogin.unilogin-component-notice-wrapper
+  @media only screen and (max-width: 600px)
+    background-color: #fff
+    position: absolute
+    bottom: 0
+    background: #8884A7
+    opacity: 1
+
+  &:before
+    @media only screen and (max-width: 600px)
+      background-color: #fff
```

Comment:
Do we need these few lines? Background-color is already set to #fff

Reply:
Yes, because we should change :before's background-color to white as well.